proxy/healthy_endpoints.go

```diff
@@ -21,7 +21,8 @@ func (h *healthyEndpoints) filterHealthyEndpoints(endpoints []routing.LBEndpoint
 	filtered := make([]routing.LBEndpoint, 0, len(endpoints))
 	for _, e := range endpoints {
 		if p < e.Metrics.HealthCheckDropProbability() {
-			/* drop */
+			ctx.Logger().Infof("dropping endpoint %q due to passive health check: p=%0.2f, dropProbability=%0.2f",
```

Comment:
maybe use normal logrus? No need to pass it I mean

Reply:
I still would like to have `ctx.Logger()` here: after https://github.com/zalando/skipper/pull/2269/ it has useful feature, it adds traceId inside log entry. It may be very helpful to see the relation between trace and log entry during debugging.